### DISCUSSION TITLE
Let helloSystem run on Chromebook

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#536] [Ideas]
I don't have a Chromebook at hand, but as long as the average thread or memory usage of helloSystem system execution is as average as Windows 7 Starter, I will think there is potential.

**Comment by probonopd** [comment]
I don't have a Chromebook at hand either, but if the Chromebook isn't locked down it might work.

**Reply by louies0623** [reply]
https://www.maketecheasier.com/replace-chromebook-bios/
This is what I can find, but I can't read it myself, maybe you can understand it better.

**Comment by louies0623** [comment]
I have recently been thinking that because it is a "Notebook" for Linux (Chrome Book), the "Notebook" provided by the manufacturer is equipped with a very low-speed CPU, so Python may occupy too much of the CPU. If we do not need it in some For calculation-related programs, replace the Python language with a language that does not require too much CPU usage, such as rust. Maybe it can be used as a test for experimental programs.

**Reply by probonopd** [reply]
Chromebooks seem to exist with low- to mid-end CPUs. But they are locked down in ways that make it harder to run third-party operating systems than on normal computers, so I never bothered to use them.

**Reply by louies0623** [reply]
It is true that Chromebooks have a special "lock", so don't bother dealing with it for now. But on low-end laptops that are not Chromebooks, can we use this system as an opportunity for them?

**Reply by probonopd** [reply]
Well, I've been testing helloSystem on an Acer TravelMate B117 with an Intel Celeron N3050.

**Reply by louies0623** [reply]
My low-end laptop is the ASUS Eee PC R105D Intel Atom N455. Since it can support 64bit, I used it for testing. I also hope that it can be useful again.

Add a bad idea: let Celeron D 331 execute helloSystem using Libreoffice, if possible

**Reply by louies0623** [reply]
I recently found a project that describes how to "unlock" Chromebooks. This is a reference.
https://github.com/SolidHal/PrawnOS#enable-developer-mode
